reactDashboard/src/components/GeoHeatMap/GeoHeatMap.css

```diff
@@ -41,17 +43,21 @@ div.leaflet-tooltip p {
 }
 
 div.leaflet-tooltip-left::before {
-  border-left-color: #000000;
+  border-left-color: #ac3232;
 }
 
 div.leaflet-tooltip-right::before {
-  border-right-color: #000000;
+  border-right-color: #071d75;
```

Comment:
changed to blue - I'm not sure what this is for

Reply:
Same here

Reply:
Without it, the triangle is white, when the tooltip is inverted horizontally.